internal/distro/fedora/images.go

```diff
@@ -381,6 +383,9 @@ func iotRawImage(workload workload.Workload,
 	if strings.HasPrefix(distro.Name(), "fedora") && !common.VersionLessThan(distro.Releasever(), "37") {
 		img.SysrootReadOnly = true
 	}
+	if strings.HasPrefix(distro.Name(), "fedora") && !common.VersionLessThan(distro.Releasever(), "38") {
+		img.Ignition = true
+	}
```

Comment:
below this, you need something like:
```
	if bpIgnition := customizations.GetIgnition(); bpIgnition != nil && bpIgnition.FirstBoot != nil && bpIgnition.FirstBoot.ProvisioningURL != "" {
		img.KernelOptionsAppend = append(img.KernelOptionsAppend, "ignition.config.url="+bpIgnition.FirstBoot.ProvisioningURL)
	}

	// 92+ only
	if kopts := customizations.GetKernel(); kopts != nil && kopts.Append != "" {
		img.KernelOptionsAppend = append(img.KernelOptionsAppend, kopts.Append)
	}
```

refer to internal/distro/rhel9/images.go

Reply:
took the liberty to clean this up @paulwhalen